### PR TITLE
[Truffle] Testing JRuby against TruffleTCK to verify consistency with…

### PR DESCRIPTION
**Pull request by jtulach**
… other languages.

### CODE REVIEW

**Comment by chrisseaton** [comment]
I don't think this is actually run by anything yet - we'll need to set up various CI systems to run them. What command did you use?

**Comment by jtulach** [comment]
When I do:
$ mvn install
the RubyTckTest gets executed as part of the build

File: truffle/pom.xml [hunk 1]
```diff
@@ -32,6 +32,17 @@
...
+      <groupId>com.oracle</groupId>
+      <artifactId>truffle-tck</artifactId>
+      <version>0.8-ef8c90391f1ec21bcfb6d5c6fb39757b860d4973-SNAPSHOT</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
```
Comment (mkristian) [inline truffle/pom.xml]: please make pom changes in pom.rb - the pom.xml are generated (and will get a big warning soon - waiting for a release for this)

**Comment by jtulach** [comment]
The changes are likely to be
```
$ git diff
diff --git a/truffle/pom.rb b/truffle/pom.rb
index e6f3e6c..a5e0df4 100644
--- a/truffle/pom.rb
+++ b/truffle/pom.rb
@@ -18,6 +18,8 @@ project 'JRuby Truffle' do
   truffle_version = '0.8-ef8c90391f1ec21bcfb6d5c6fb39757b860d4973-SNAPSHOT'
   jar 'com.oracle:truffle:' + truffle_version
   jar 'com.oracle:truffle-dsl-processor:' + truffle_version, :scope => 'provided'
+  jar 'com.oracle:truffle-tck:' + truffle_version, :scope => 'test'
+  jar 'junit:junit', :scope => 'test'
```
but I am not sure how to verify that. My build seems to be using pom.xml all the time.

**Comment by mkristian** [comment]
use mvn-3.3.x then it will pick the pom.rb over pom.xml. and there is ```mvnw``` in the root directory which is basically mvn-3.3.3

mvn-3.2.x or older will pick the pom.xml, like IDEs, etc

**Comment by eregon** [comment]
What is "TruffleTCK" ?

**Comment by chrisseaton** [comment]
http://mail.openjdk.java.net/pipermail/graal-dev/2015-June/003233.html

**Pull request merged**